webapp/channels/src/components/user_list_row_with_error/user_list_row_with_error.tsx

```diff
@@ -90,10 +88,11 @@ export default class UserListRowWithError extends React.PureComponent<Props, Sta
             email = null;
         } else if (this.props.extraInfo && this.props.extraInfo.length > 0) {
             email = (
-                <FormattedMarkdownMessage
-                    id='admin.user_item.emailTitle'
-                    defaultMessage='**Email:** {email}'
+                <FormattedMessage
+                    id='admin.user_item.email_title'
+                    defaultMessage='<strong>Email:</strong> {email}'
```

Comment:
lets use `b` tag here since email here doesnt have to be bold in structure point of view

Reply:
Okay, I will update it to use b tag.

Reply:
Hello @M-ZubairAhmed , I have updated to use `b` tag. Please re-review.